### PR TITLE
Add batch review form

### PR DESCRIPTION
**Pull request by kewitham**
[#1691 Implement Disposition Batching / Approval: Records Team](https://github.com/usdoj-crt/crt-portal-management/issues/1691)

## What does this change?
This PR adds a form to allow the records team to review disposal batches.
Future PRs will add a distinction between the second and first reviewer's notes, logging, pagination to the records list and permissions for batch review
## Screenshots (for front-end PR):
Image
### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [ ] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [ ] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/forms.py [hunk 1]
```diff
@@ -2028,6 +2028,167 @@ class PrintActions(Form):
...
+        )
+
+    def clean_first_reviewer(self):
+        user = self.user.get_username()
+        if 'first_reviewer' not in self.cleaned_data or self.instance.first_reviewer is not None:
+            return self.instance.first_reviewer
+        name = self.cleaned_data['first_reviewer'].split(' ')
+        if len(name) == 2:
+            user = User.objects.filter(first_name=name[0], last_name=name[1]).first()
+        return user
```
Comment (okkays) [inline crt_portal/cts_forms/forms.py:2155]: oh interesting - when we submit this form, we can't have the display be the user's name but the value be the user's id?

for example, there's definitely two amanda sanchez at DOJ

File: crt_portal/cts_forms/views.py [hunk 2]
```diff
@@ -1184,16 +1188,67 @@ class DispositionBatchActionsView(LoginRequiredMixin, FormView):
...
+            return redirect(f"{url}{return_url_args}")
+        else:
+            for key in form.errors:
+                errors = '; '.join(form.errors[key])
+                if key == '__all__':
+                    target = ':'
+                else:
+                    target = f' {key}:'
+                error_message = f'Could not batch reports{target} {errors}'
+                messages.add_message(request, messages.ERROR, error_message)
```
Comment (okkays) [inline crt_portal/cts_forms/views.py:1233]:
```suggestion
          for key in form.errors:
              errors = '; '.join(form.errors[key])
              if key == '__all__':
                  target = ':'
              else:
                  target = f' {key}:'
              error_message = f'Could not batch reports{target} {errors}'
              messages.add_message(request, messages.ERROR, error_message)
```

**Review by okkays — APPROVED** [review]

**Review by okkays — CHANGES_REQUESTED** [review]
Sorry, meant to request changes to this one with the user id note

File: crt_portal/cts_forms/forms.py (hunk 1 above)
Comment (kewitham) [reply crt_portal/cts_forms/forms.py]: Good catch! Updated this to use the user id

**Review by okkays — APPROVED** [review]
nice, lgtm!

**Pull request merged**